src/lib/utils/hooks/useTraceUpdate.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useRef } from 'react'
```

Comment:
I didn't mean to commit this, but can be handy for debugging, so I'm tempted to leave it.

Reply:
leave it!